lib/plugins/blocks.js

```diff
@@ -458,24 +458,33 @@ function inject (bot, { version, storageBuilder }) {
     startListenerProxy()
   })
 
+  let listenerState = false
```

Comment:
Why not store the listener instead ? It would be cleaner

Reply:
Could you give an example of storing the listener?

Is it like putting a variable before the listener?

Reply:
I believe this is what you mean.